generic/kubernetes/single-region/tests/helm-values/registry.yml

```diff
@@ -0,0 +1,20 @@
+---
+# This file contains specific values used during the integration tests
+
+
+# Auth to avoid Docker download rate limit.
+# https://docs.docker.com/docker-hub/download-rate-limit/
+identityKeycloak:
+    image:
+        pullSecrets:
+            - name: index-docker-io
```

Comment:
maybe it's hidden in one of the other files but where do we actually define `index-docker-io`?